src/main/resources/spring.gradle

```diff
@@ -7,10 +7,20 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-validation'
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    //===start===
+    //  Spring Cloud Sleuth for distributed tracing with Open Telemetry support
+    implementation('org.springframework.cloud:spring-cloud-starter-sleuth') {
+        exclude group: 'org.springframework.cloud', module: 'spring-cloud-sleuth-brave'
+    } //  We want to exclude the default tracer coming from Sleuth...
+    // ... and we want to include the one coming from Spring Cloud Sleuth OTel
```

Comment:
I think the actual reason for including OTel is because OpenTracing is archived project now and people are asked to upgrade to OpenTelementry

Reply:
actually spring cloud sleuth by default includes exporting using brave (compatible with zipkin way of gathering traces)
that's why we are excluding it 
then we could add some of the others supported exporters - jaeger, opentracing(deprecated one), otel and so on.
We choose otel - because it is a standard already supported by most of the backend tracing solutions(even if it is supported using additional agent)